lib/commands/init.mjs

```diff
@@ -3,6 +3,19 @@ import fs from 'fs';
 import path from 'path';
 import ejs from 'ejs';
 import {getAbsolutePathFromRelativeToRoot} from "../utils/path_utils.mjs";
+import {isSystemDirectory} from "../commands/pathdetect.mjs";
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+let isSDirectory = isSystemDirectory(__dirname);
+console.log(isSDirectory);
```

Comment:
We probably don't want to keep this in the output, so we should clean that line up.